src/proof/dot/dot_printer.cpp

```diff
@@ -348,7 +348,7 @@ void DotPrinter::printProofNodeInfo(std::ostream& out, const ProofNode* pn)
 
   out << "\t" << d_ruleID << " [ label = \"{";
 
-  resultStr << d_lbind.convert(pn->getResult(), "let");
+  resultStr << d_lbind.convert(pn->getResult();
```

Comment:
```suggestion
  resultStr << d_lbind.convert(pn->getResult());
```
Sorry, I introduced a typo here before.

Reply:
Applied and resquashed